2012-11-25  Adam Conrad  <adconrad@0c3.net>

	* sysdeps/unix/sysv/linux/powerpc/sys/procfs.h: Support
	_UAPI_ASM_POWERPC_ELF_H in the Linux 3.7 headers.

More info at: http://article.gmane.org/gmane.linux.kernel.cross-arch/15847
              http://sourceware.org/ml/libc-alpha/2012-11/msg00673.html

diff --git a/sysdeps/unix/sysv/linux/powerpc/sys/procfs.h b/sysdeps/unix/sysv/linux/powerpc/sys/procfs.h
index dc41bdf..a8f9878 100644
--- a/sysdeps/unix/sysv/linux/powerpc/sys/procfs.h
+++ b/sysdeps/unix/sysv/linux/powerpc/sys/procfs.h
@@ -34,7 +34,9 @@ __BEGIN_DECLS
 /* These definitions are normally provided by ucontext.h via 
    asm/sigcontext.h, asm/ptrace.h, and asm/elf.h.  Otherwise we define 
    them here.  */ 
-#if !defined __PPC64_ELF_H && !defined _ASM_POWERPC_ELF_H
+#if !defined __PPC64_ELF_H \
+    && !defined _ASM_POWERPC_ELF_H \
+    && !defined _UAPI_ASM_POWERPC_ELF_H
 #define ELF_NGREG       48      /* includes nip, msr, lr, etc. */
 #define ELF_NFPREG      33      /* includes fpscr */
 #if __WORDSIZE == 32
